### ISSUE TITLE
bug: OTP verification failing, errors in logs

### ISSUE THREAD

**Issue by mahenkpatel** [rabilrbl/jiotv_go#296]
### Bug description

Today, when I tried to verify using OTP its failing even after entering correct otp multiple times. The logs are full of errors related to panic (added below)

### Steps to reproduce

Try to verify using otp, verification fails

### Relevant log output

```shell
panic: Response:  {"code":400,"message":"Parameter Missing/Bad Request"}

goroutine 44 [running]:
runtime/debug.Stack()
        runtime/debug/stack.go:24 +0x5e
github.com/gofiber/fiber/v2/middleware/recover.defaultStackTraceHandler(0xc0002af010?, {0x927320?, 0xc00029b130})
        github.com/gofiber/fiber/v2@v2.52.4/middleware/recover/recover.go:12 +0x26
github.com/rabilrbl/jiotv_go/v3/cmd.JioTVServer.New.func1.1()
        github.com/gofiber/fiber/v2@v2.52.4/middleware/recover/recover.go:31 +0x72
panic({0x927320?, 0xc00029b130?})
        runtime/panic.go:914 +0x21f
log.(*Logger).Panicln(0xc0018388c0?, {0xc0002af2b8?, 0x44?, 0xc0002af2d8?})
        log/log.go:317 +0x65
github.com/rabilrbl/jiotv_go/v3/pkg/television.(*Television).Live(0xc00008c640, {0xc0053c7b80, 0x4})
        github.com/rabilrbl/jiotv_go/v3/pkg/television/television.go:129 +0x9ba
github.com/rabilrbl/jiotv_go/v3/internal/handlers.LiveHandler(0xc001778000)
        github.com/rabilrbl/jiotv_go/v3/internal/handlers/handlers.go:134 +0x75
github.com/gofiber/fiber/v2.(*App).next(0xc000231900, 0xc001778000)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:145 +0x1b2
github.com/gofiber/fiber/v2.(*Ctx).Next(0x5?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1030 +0x4d
github.com/gofiber/fiber/v2/middleware/etag.New.func1(0xc001778000)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/etag/etag.go:33 +0xb5
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc001778000?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1027 +0x3d
github.com/gofiber/fiber/v2/middleware/helmet.New.func1(0xc001778000)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/helmet/helmet.go:92 +0x53f
github.com/gofiber/fiber/v2.(*App).next(0xc000231900, 0xc001778000)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:145 +0x1b2
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc00029e480?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1030 +0x4d
github.com/gofiber/fiber/v2/middleware/logger.New.func3(0xc001778000)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/logger/logger.go:119 +0x316
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc0053c6bd0?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1027 +0x3d
github.com/rabilrbl/jiotv_go/v3/cmd.JioTVServer.CORS.func2(0xf?)
        github.com/rabilrbl/jiotv_go/v3/internal/middleware/cors.go:30 +0x147
github.com/gofiber/fiber/v2.(*Ctx).Next(0x7e68bc?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1027 +0x3d
github.com/rabilrbl/jiotv_go/v3/cmd.JioTVServer.New.func1(0x94ab20?)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/recover/recover.go:43 +0xbb
github.com/gofiber/fiber/v2.(*App).next(0xc000231900, 0xc001778000)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:145 +0x1b2
github.com/gofiber/fiber/v2.(*App).handler(0xc000231900, 0x4c790f?)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:172 +0x78
github.com/valyala/fasthttp.(*Server).serveConn(0xc00013a900, {0xbf4638?, 0xc0002a2058})
        github.com/valyala/fasthttp@v1.52.0/server.go:2374 +0x11d4
github.com/valyala/fasthttp.(*workerPool).workerFunc(0xc000344000, 0xc000294380)
        github.com/valyala/fasthttp@v1.52.0/workerpool.go:224 +0xa4
github.com/valyala/fasthttp.(*workerPool).getCh.func1()
        github.com/valyala/fasthttp@v1.52.0/workerpool.go:196 +0x32
created by github.com/valyala/fasthttp.(*workerPool).getCh in goroutine 1
        github.com/valyala/fasthttp@v1.52.0/workerpool.go:195 +0x1ab

panic: Response:  {"code":400,"message":"Parameter Missing/Bad Request"}

goroutine 44 [running]:
runtime/debug.Stack()
        runtime/debug/stack.go:24 +0x5e
github.com/gofiber/fiber/v2/middleware/recover.defaultStackTraceHandler(0xc0002af010?, {0x927320?, 0xc0025a29a0})
        github.com/gofiber/fiber/v2@v2.52.4/middleware/recover/recover.go:12 +0x26
github.com/rabilrbl/jiotv_go/v3/cmd.JioTVServer.New.func1.1()
        github.com/gofiber/fiber/v2@v2.52.4/middleware/recover/recover.go:31 +0x72
panic({0x927320?, 0xc0025a29a0?})
        runtime/panic.go:914 +0x21f
log.(*Logger).Panicln(0xc00188c500?, {0xc0002af2b8?, 0x44?, 0xc0002af2d8?})
        log/log.go:317 +0x65
github.com/rabilrbl/jiotv_go/v3/pkg/television.(*Television).Live(0xc00008c640, {0xc0052ef9e8, 0x4})
        github.com/rabilrbl/jiotv_go/v3/pkg/television/television.go:129 +0x9ba
github.com/rabilrbl/jiotv_go/v3/internal/handlers.LiveHandler(0xc000004300)
        github.com/rabilrbl/jiotv_go/v3/internal/handlers/handlers.go:134 +0x75
github.com/gofiber/fiber/v2.(*App).next(0xc000231900, 0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:145 +0x1b2
github.com/gofiber/fiber/v2.(*Ctx).Next(0x5?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1030 +0x4d
github.com/gofiber/fiber/v2/middleware/etag.New.func1(0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/etag/etag.go:33 +0xb5
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc000004300?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1027 +0x3d
github.com/gofiber/fiber/v2/middleware/helmet.New.func1(0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/helmet/helmet.go:92 +0x53f
github.com/gofiber/fiber/v2.(*App).next(0xc000231900, 0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:145 +0x1b2
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc00029e480?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1030 +0x4d
github.com/gofiber/fiber/v2/middleware/logger.New.func3(0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/logger/logger.go:119 +0x316
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc0052ee030?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1027 +0x3d
github.com/rabilrbl/jiotv_go/v3/cmd.JioTVServer.CORS.func2(0xf?)
        github.com/rabilrbl/jiotv_go/v3/internal/middleware/cors.go:30 +0x147
github.com/gofiber/fiber/v2.(*Ctx).Next(0x7e68bc?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1027 +0x3d
github.com/rabilrbl/jiotv_go/v3/cmd.JioTVServer.New.func1(0x94ab20?)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/recover/recover.go:43 +0xbb
github.com/gofiber/fiber/v2.(*App).next(0xc000231900, 0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:145 +0x1b2
github.com/gofiber/fiber/v2.(*App).handler(0xc000231900, 0x4c790f?)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:172 +0x78
github.com/valyala/fasthttp.(*Server).serveConn(0xc00013a900, {0xbf4638?, 0xc0002a2058})
        github.com/valyala/fasthttp@v1.52.0/server.go:2374 +0x11d4
github.com/valyala/fasthttp.(*workerPool).workerFunc(0xc000344000, 0xc000294380)
        github.com/valyala/fasthttp@v1.52.0/workerpool.go:224 +0xa4
github.com/valyala/fasthttp.(*workerPool).getCh.func1()
        github.com/valyala/fasthttp@v1.52.0/workerpool.go:196 +0x32
created by github.com/valyala/fasthttp.(*workerPool).getCh in goroutine 1
        github.com/valyala/fasthttp@v1.52.0/workerpool.go:195 +0x1ab

panic: Response:  {"code":400,"message":"Parameter Missing/Bad Request"}

goroutine 44 [running]:
runtime/debug.Stack()
        runtime/debug/stack.go:24 +0x5e
github.com/gofiber/fiber/v2/middleware/recover.defaultStackTraceHandler(0xc0002af010?, {0x927320?, 0xc00029b190})
        github.com/gofiber/fiber/v2@v2.52.4/middleware/recover/recover.go:12 +0x26
github.com/rabilrbl/jiotv_go/v3/cmd.JioTVServer.New.func1.1()
        github.com/gofiber/fiber/v2@v2.52.4/middleware/recover/recover.go:31 +0x72
panic({0x927320?, 0xc00029b190?})
        runtime/panic.go:914 +0x21f
log.(*Logger).Panicln(0xc001838900?, {0xc0002af2b8?, 0x44?, 0xc0002af2d8?})
        log/log.go:317 +0x65
github.com/rabilrbl/jiotv_go/v3/pkg/television.(*Television).Live(0xc00008c640, {0xc0052efa70, 0x4})
        github.com/rabilrbl/jiotv_go/v3/pkg/television/television.go:129 +0x9ba
github.com/rabilrbl/jiotv_go/v3/internal/handlers.LiveHandler(0xc001778000)
        github.com/rabilrbl/jiotv_go/v3/internal/handlers/handlers.go:134 +0x75
github.com/gofiber/fiber/v2.(*App).next(0xc000231900, 0xc001778000)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:145 +0x1b2
github.com/gofiber/fiber/v2.(*Ctx).Next(0x5?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1030 +0x4d
github.com/gofiber/fiber/v2/middleware/etag.New.func1(0xc001778000)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/etag/etag.go:33 +0xb5
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc001778000?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1027 +0x3d
github.com/gofiber/fiber/v2/middleware/helmet.New.func1(0xc001778000)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/helmet/helmet.go:92 +0x53f
github.com/gofiber/fiber/v2.(*App).next(0xc000231900, 0xc001778000)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:145 +0x1b2
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc00029e480?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1030 +0x4d
github.com/gofiber/fiber/v2/middleware/logger.New.func3(0xc001778000)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/logger/logger.go:119 +0x316
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc0053c6bd0?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1027 +0x3d
github.com/rabilrbl/jiotv_go/v3/cmd.JioTVServer.CORS.func2(0xf?)
        github.com/rabilrbl/jiotv_go/v3/internal/middleware/cors.go:30 +0x147
github.com/gofiber/fiber/v2.(*Ctx).Next(0x7e68bc?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1027 +0x3d
github.com/rabilrbl/jiotv_go/v3/cmd.JioTVServer.New.func1(0x94ab20?)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/recover/recover.go:43 +0xbb
github.com/gofiber/fiber/v2.(*App).next(0xc000231900, 0xc001778000)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:145 +0x1b2
github.com/gofiber/fiber/v2.(*App).handler(0xc000231900, 0x4c790f?)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:172 +0x78
github.com/valyala/fasthttp.(*Server).serveConn(0xc00013a900, {0xbf4638?, 0xc0002a2058})
        github.com/valyala/fasthttp@v1.52.0/server.go:2374 +0x11d4
github.com/valyala/fasthttp.(*workerPool).workerFunc(0xc000344000, 0xc000294380)
        github.com/valyala/fasthttp@v1.52.0/workerpool.go:224 +0xa4
github.com/valyala/fasthttp.(*workerPool).getCh.func1()
        github.com/valyala/fasthttp@v1.52.0/workerpool.go:196 +0x32
created by github.com/valyala/fasthttp.(*workerPool).getCh in goroutine 1
        github.com/valyala/fasthttp@v1.52.0/workerpool.go:195 +0x1ab

panic: Response:  {"code":400,"message":"Parameter Missing/Bad Request"}

goroutine 44 [running]:
runtime/debug.Stack()
        runtime/debug/stack.go:24 +0x5e
github.com/gofiber/fiber/v2/middleware/recover.defaultStackTraceHandler(0xc0002af010?, {0x927320?, 0xc0025a2a50})
        github.com/gofiber/fiber/v2@v2.52.4/middleware/recover/recover.go:12 +0x26
github.com/rabilrbl/jiotv_go/v3/cmd.JioTVServer.New.func1.1()
        github.com/gofiber/fiber/v2@v2.52.4/middleware/recover/recover.go:31 +0x72
panic({0x927320?, 0xc0025a2a50?})
        runtime/panic.go:914 +0x21f
log.(*Logger).Panicln(0xc00188c540?, {0xc0002af2b8?, 0x44?, 0xc0002af2d8?})
        log/log.go:317 +0x65
github.com/rabilrbl/jiotv_go/v3/pkg/television.(*Television).Live(0xc00008c640, {0xc0053c7c88, 0x4})
        github.com/rabilrbl/jiotv_go/v3/pkg/television/television.go:129 +0x9ba
github.com/rabilrbl/jiotv_go/v3/internal/handlers.LiveHandler(0xc000004300)
        github.com/rabilrbl/jiotv_go/v3/internal/handlers/handlers.go:134 +0x75
github.com/gofiber/fiber/v2.(*App).next(0xc000231900, 0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:145 +0x1b2
github.com/gofiber/fiber/v2.(*Ctx).Next(0x5?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1030 +0x4d
github.com/gofiber/fiber/v2/middleware/etag.New.func1(0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/etag/etag.go:33 +0xb5
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc000004300?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1027 +0x3d
github.com/gofiber/fiber/v2/middleware/helmet.New.func1(0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/helmet/helmet.go:92 +0x53f
github.com/gofiber/fiber/v2.(*App).next(0xc000231900, 0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:145 +0x1b2
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc00029e480?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1030 +0x4d
github.com/gofiber/fiber/v2/middleware/logger.New.func3(0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/logger/logger.go:119 +0x316
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc0052ee030?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1027 +0x3d
github.com/rabilrbl/jiotv_go/v3/cmd.JioTVServer.CORS.func2(0xf?)
        github.com/rabilrbl/jiotv_go/v3/internal/middleware/cors.go:30 +0x147
github.com/gofiber/fiber/v2.(*Ctx).Next(0x7e68bc?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1027 +0x3d
github.com/rabilrbl/jiotv_go/v3/cmd.JioTVServer.New.func1(0x94ab20?)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/recover/recover.go:43 +0xbb
github.com/gofiber/fiber/v2.(*App).next(0xc000231900, 0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:145 +0x1b2
github.com/gofiber/fiber/v2.(*App).handler(0xc000231900, 0x4c790f?)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:172 +0x78
github.com/valyala/fasthttp.(*Server).serveConn(0xc00013a900, {0xbf4638?, 0xc0002a2058})
        github.com/valyala/fasthttp@v1.52.0/server.go:2374 +0x11d4
github.com/valyala/fasthttp.(*workerPool).workerFunc(0xc000344000, 0xc000294380)
        github.com/valyala/fasthttp@v1.52.0/workerpool.go:224 +0xa4
github.com/valyala/fasthttp.(*workerPool).getCh.func1()
        github.com/valyala/fasthttp@v1.52.0/workerpool.go:196 +0x32
created by github.com/valyala/fasthttp.(*workerPool).getCh in goroutine 1
        github.com/valyala/fasthttp@v1.52.0/workerpool.go:195 +0x1ab

panic: Response:  {"code":400,"message":"Parameter Missing/Bad Request"}

goroutine 44 [running]:
runtime/debug.Stack()
        runtime/debug/stack.go:24 +0x5e
github.com/gofiber/fiber/v2/middleware/recover.defaultStackTraceHandler(0xc0002af010?, {0x927320?, 0xc0025a2ac0})
        github.com/gofiber/fiber/v2@v2.52.4/middleware/recover/recover.go:12 +0x26
github.com/rabilrbl/jiotv_go/v3/cmd.JioTVServer.New.func1.1()
        github.com/gofiber/fiber/v2@v2.52.4/middleware/recover/recover.go:31 +0x72
panic({0x927320?, 0xc0025a2ac0?})
        runtime/panic.go:914 +0x21f
log.(*Logger).Panicln(0xc00188c580?, {0xc0002af2b8?, 0x44?, 0xc0002af2d8?})
        log/log.go:317 +0x65
github.com/rabilrbl/jiotv_go/v3/pkg/television.(*Television).Live(0xc00008c640, {0xc0053c7d40, 0x4})
        github.com/rabilrbl/jiotv_go/v3/pkg/television/television.go:129 +0x9ba
github.com/rabilrbl/jiotv_go/v3/internal/handlers.LiveHandler(0xc000004300)
        github.com/rabilrbl/jiotv_go/v3/internal/handlers/handlers.go:134 +0x75
github.com/gofiber/fiber/v2.(*App).next(0xc000231900, 0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:145 +0x1b2
github.com/gofiber/fiber/v2.(*Ctx).Next(0x5?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1030 +0x4d
github.com/gofiber/fiber/v2/middleware/etag.New.func1(0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/etag/etag.go:33 +0xb5
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc000004300?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1027 +0x3d
github.com/gofiber/fiber/v2/middleware/helmet.New.func1(0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/helmet/helmet.go:92 +0x53f
github.com/gofiber/fiber/v2.(*App).next(0xc000231900, 0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:145 +0x1b2
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc00029e480?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1030 +0x4d
github.com/gofiber/fiber/v2/middleware/logger.New.func3(0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/logger/logger.go:119 +0x316
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc0052ee030?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1027 +0x3d
github.com/rabilrbl/jiotv_go/v3/cmd.JioTVServer.CORS.func2(0xf?)
        github.com/rabilrbl/jiotv_go/v3/internal/middleware/cors.go:30 +0x147
github.com/gofiber/fiber/v2.(*Ctx).Next(0x7e68bc?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1027 +0x3d
github.com/rabilrbl/jiotv_go/v3/cmd.JioTVServer.New.func1(0x94ab20?)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/recover/recover.go:43 +0xbb
github.com/gofiber/fiber/v2.(*App).next(0xc000231900, 0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:145 +0x1b2
github.com/gofiber/fiber/v2.(*App).handler(0xc000231900, 0x4c790f?)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:172 +0x78
github.com/valyala/fasthttp.(*Server).serveConn(0xc00013a900, {0xbf4638?, 0xc0002a2058})
        github.com/valyala/fasthttp@v1.52.0/server.go:2374 +0x11d4
github.com/valyala/fasthttp.(*workerPool).workerFunc(0xc000344000, 0xc000294380)
        github.com/valyala/fasthttp@v1.52.0/workerpool.go:224 +0xa4
github.com/valyala/fasthttp.(*workerPool).getCh.func1()
        github.com/valyala/fasthttp@v1.52.0/workerpool.go:196 +0x32
created by github.com/valyala/fasthttp.(*workerPool).getCh in goroutine 1
        github.com/valyala/fasthttp@v1.52.0/workerpool.go:195 +0x1ab

panic: Response:  {"code":400,"message":"Parameter Missing/Bad Request"}

goroutine 44 [running]:
runtime/debug.Stack()
        runtime/debug/stack.go:24 +0x5e
github.com/gofiber/fiber/v2/middleware/recover.defaultStackTraceHandler(0xc0002af010?, {0x927320?, 0xc0025a2b10})
        github.com/gofiber/fiber/v2@v2.52.4/middleware/recover/recover.go:12 +0x26
github.com/rabilrbl/jiotv_go/v3/cmd.JioTVServer.New.func1.1()
        github.com/gofiber/fiber/v2@v2.52.4/middleware/recover/recover.go:31 +0x72
panic({0x927320?, 0xc0025a2b10?})
        runtime/panic.go:914 +0x21f
log.(*Logger).Panicln(0xc00188c5c0?, {0xc0002af2b8?, 0x44?, 0xc0002af2d8?})
        log/log.go:317 +0x65
github.com/rabilrbl/jiotv_go/v3/pkg/television.(*Television).Live(0xc00008c640, {0xc0053c7dd8, 0x4})
        github.com/rabilrbl/jiotv_go/v3/pkg/television/television.go:129 +0x9ba
github.com/rabilrbl/jiotv_go/v3/internal/handlers.LiveHandler(0xc000004300)
        github.com/rabilrbl/jiotv_go/v3/internal/handlers/handlers.go:134 +0x75
github.com/gofiber/fiber/v2.(*App).next(0xc000231900, 0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:145 +0x1b2
github.com/gofiber/fiber/v2.(*Ctx).Next(0x5?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1030 +0x4d
github.com/gofiber/fiber/v2/middleware/etag.New.func1(0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/etag/etag.go:33 +0xb5
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc000004300?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1027 +0x3d
github.com/gofiber/fiber/v2/middleware/helmet.New.func1(0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/helmet/helmet.go:92 +0x53f
github.com/gofiber/fiber/v2.(*App).next(0xc000231900, 0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:145 +0x1b2
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc00029e480?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1030 +0x4d
github.com/gofiber/fiber/v2/middleware/logger.New.func3(0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/logger/logger.go:119 +0x316
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc0052ee030?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1027 +0x3d
github.com/rabilrbl/jiotv_go/v3/cmd.JioTVServer.CORS.func2(0xf?)
        github.com/rabilrbl/jiotv_go/v3/internal/middleware/cors.go:30 +0x147
github.com/gofiber/fiber/v2.(*Ctx).Next(0x7e68bc?)
        github.com/gofiber/fiber/v2@v2.52.4/ctx.go:1027 +0x3d
github.com/rabilrbl/jiotv_go/v3/cmd.JioTVServer.New.func1(0x94ab20?)
        github.com/gofiber/fiber/v2@v2.52.4/middleware/recover/recover.go:43 +0xbb
github.com/gofiber/fiber/v2.(*App).next(0xc000231900, 0xc000004300)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:145 +0x1b2
github.com/gofiber/fiber/v2.(*App).handler(0xc000231900, 0x4c790f?)
        github.com/gofiber/fiber/v2@v2.52.4/router.go:172 +0x78
github.com/valyala/fasthttp.(*Server).serveConn(0xc00013a900, {0xbf4638?, 0xc0002a2058})
        github.com/valyala/fasthttp@v1.52.0/server.go:2374 +0x11d4
github.com/valyala/fasthttp.(*workerPool).workerFunc(0xc000344000, 0xc000294380)
        github.com/valyala/fasthttp@v1.52.0/workerpool.go:224 +0xa4
github.com/valyala/fasthttp.(*workerPool).getCh.func1()
        github.com/valyala/fasthttp@v1.52.0/workerpool.go:196 +0x32
created by github.com/valyala/fasthttp.(*workerPool).getCh in goroutine 1
        github.com/valyala/fasthttp@v1.52.0/workerpool.go:195 +0x1ab

panic: Response:  {"code":400,"message":"Parameter Missing/Bad Request"}
```

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by rabilrbl** [comment]
@mahenkpatel 
Please mention your OS, 
Internet provider, if possible and CPU architecture

**Comment by mahenkpatel** [comment]
Installed on docker on Ubuntu 22.04 on a Intel 5505 mini PC, x64

Airtel ISP, everything else works fine. Jiotv go was also working fine till yesterday

OTP gets generated but when its entered on Jiotv go the error shows as otp verification failed.

**Comment by Gopsy1007** [comment]
Facing same issue on my raspberry Pi 4 docker container

**Comment by rabilrbl** [comment]
@mahenkpatel @Gopsy1007 
For now this the temporary fix
Delete store.toml from .jiotv_go folder in your user directory

**Comment by Gopsy1007** [comment]
> @mahenkpatel @Gopsy1007 
> For now this the temporary fix
> Delete `store.toml` from `.jiotv_go` folder in your user directory

Thanks

**Comment by rabilrbl** [comment]
@mahenkpatel @Gopsy1007 
As soon as u experience this error.
Please upload entire jiotv_go.log at https://telegram.me/jiotv_go_bot

More info: https://telegram.me/jiotv_go/64

**Comment by rabilrbl** [comment]
@mahenkpatel Please respond if the issue is fixed with JioTV Go v3.5

Issue will be closed as stale, if no response.
